"""
Написать функцию, которая принимает на вход число от 1 до 100.
Если число равно 13, функция поднимает исключительную ситуации ValueError
иначе возвращает введенное число, возведенное в квадрат.
Далее написать основной код программы. Пользователь вводит число.
Введенное число передаем параметром в написанную функцию и печатаем результат,
который вернула функция. Обработать возможность возникновения исключительной ситуации,
которая поднимается внутри функции.
"""


def magic(number):
    if number == 13:
        raise ValueError

    return number ** 2


try:
    result = magic(int(input('Введите число: ')))
except ValueError:
    print('Ошибка')
else:
    print(result)
